bird.js

```diff
@@ -3,6 +3,9 @@
 // http://patreon.com/codingtrain
 // Code for: https://youtu.be/cXgA1d_E-jY
 
+/* global birdSprite */
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Is there a way to do this in a config file? I would like the code to have minimal comments to reduce any noise as I use it as the basis for the neuro-evolution tutorials.

Reply:
I had been searching for a while, a possibility is to move everything into a global comment. I am currently looking at whether it can be hidden in the .eslintrc (at which point I'll run a base configuration for you to extend) but haven't found that yet.

Reply:
Apart from the fact that I think that the global asset thing is poor code-style, I don't think eslint has an easy way to configure usage checking globally. (using `"no-unused-vars": ["error", {"vars":'local"}]`) would be an option but that would miss a lot of critical linting errors. In the commit that I am pushing right now, I also replaced the disable-next-line clauses with exported clauses, as they are a bit clearer.

Reply:
I can live with the globals though I would probably prefer to put them in a config file and not have anything related to eslint in the code itself. But if having these comments is standard JavaScript practice then maybe it's good to have that here and expose viewers to this option when linting! What about also adding something that says:

```javascript
/* comments for eslint configuration */
```

There's probably a better way to phrase that. What do you think @meiamsome?

Reply:
We can add globals to the config file for simplicity in this case, but I am not sure what is the best course of action.

Reply:
I'd like to try that unless anyone objects! @rhbvkleef do you want to add that to this pul request? I'm also happy to merge and refactor after if you prefer.

Reply:
Even though I personally disagree with configurations like this, I added them as you requested.